gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -0,0 +1,226 @@
+                  <NxTable.Cell>Type</NxTable.Cell>
+                  <NxTable.Cell>Required</NxTable.Cell>
+                  <NxTable.Cell>Details</NxTable.Cell>
+                </NxTable.Row>
+              </NxTable.Head>
+              <NxTable.Body>
+                <NxTable.Row>
+                  <NxTable.Cell>isSelected</NxTable.Cell>
+                  <NxTable.Cell>Boolean</NxTable.Cell>
+                  <NxTable.Cell>Optional</NxTable.Cell>
+                  <NxTable.Cell>Toggle for the selected state of the currently clicked link</NxTable.Cell>
+                </NxTable.Row>
+                <NxTable.Row>
+                  <NxTable.Cell>icon</NxTable.Cell>
+                  <NxTable.Cell>IconDefinition</NxTable.Cell>
+                  <NxTable.Cell>Yes</NxTable.Cell>
+                  <NxTable.Cell>
+                    Navigation links have icons to their left, this specifies the icon to be used.
+                  </NxTable.Cell>
+                </NxTable.Row>
+                <NxTable.Row>
+                  <NxTable.Cell>text</NxTable.Cell>
+                  <NxTable.Cell>string</NxTable.Cell>
+                  <NxTable.Cell>Yes</NxTable.Cell>
+                  <NxTable.Cell>The text that should appear in the navigation link</NxTable.Cell>
+                </NxTable.Row>
+                <NxTable.Row>
+                  <NxTable.Cell>href</NxTable.Cell>
+                  <NxTable.Cell>string</NxTable.Cell>
+                  <NxTable.Cell>Yes</NxTable.Cell>
+                  <NxTable.Cell>URL</NxTable.Cell>
+                </NxTable.Row>
+              </NxTable.Body>
+            </NxTable>
+          </section>
+          <section className="nx-tile-subsection">
+            <header className="nx-tile-subsection__header">
+              <h3 className="nx-h3"><NxCode>NxGlobalSidebarNavigationLink</NxCode> Props</h3>
+            </header>
+            <p className="nx-p">The following CSS classes are available for use on child elements.</p>
+            <NxTable>
+              <NxTable.Head>
+                <NxTable.Row>
+                  <NxTable.Cell>Name</NxTable.Cell>
+                  <NxTable.Cell>Location</NxTable.Cell>
+                  <NxTable.Cell>Description</NxTable.Cell>
+                </NxTable.Row>
+              </NxTable.Head>
+              <NxTable.Body>
+                <NxTable.Row>
+                  <NxTable.Cell><NxCode>nx-global-sidebar__expanded-content</NxCode></NxTable.Cell>
+                  <NxTable.Cell>element</NxTable.Cell>
+                  <NxTable.Cell>
+                    Content wrapped with <NxCode>nx-global-sidebar__expanded-content</NxCode> will appear in the
+                    expanded view, but will not be displayed in the collapsed view.
+                  </NxTable.Cell>
+                </NxTable.Row>
+              </NxTable.Body>
+            </NxTable>
+          </section>
+        </div>
+      </section>
+      <section className="nx-tile">
```

Comment:
Keep in mind pretty much all of the RSC styles you're using in this file have convenience components.

Reply:
> Keep in mind pretty much all of the RSC styles you're using in this file have convenience components.

I've converted most if not all here: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/58ebce60573ddfb22d395e8a8ebca0557f73596d

For me the more complex convenience components (NxTile et.al.) are not more convenient than writing the straight HTML. As far as I'm concerned we could have stopped with the basic HTML tags. I am not in favour of replacing HTML with convenience components and would have pushed back hard were I aware that was your line of thinking.